UndertaleModLib/Models/UndertaleRoom.cs

```diff
@@ -166,20 +166,26 @@ public enum RoomEntryFlags : uint
         /// </summary>
         public UndertaleSimpleList<UndertaleResourceById<UndertaleSequence, UndertaleChunkSEQN>> Sequences { get; private set; } = new UndertaleSimpleList<UndertaleResourceById<UndertaleSequence, UndertaleChunkSEQN>>();
 
-        private Layer GetBGColorLayer()
-        {
-            return _layers?.Where(l => l.LayerType is LayerType.Background
-                                    && l.BackgroundData.Sprite is null
-                                    && l.BackgroundData.Color != 0)
-                           .OrderBy(l => l?.LayerDepth ?? 0)
-                           .FirstOrDefault();
-        }
         public void UpdateBGColorLayer() => OnPropertyChanged("BGColorLayer");
 
         /// <summary>
-        /// The layer containing the background color.
+        /// The layer containing the background color.<br/>
+        /// Used by "BGColorConverter" of the UndertaleModTool room editor.
```

Comment:
Put this into remarks please, not summary.
Also link to BGColorConverter via a cref attribute

Reply:
Okay.
I've tried to add `<see cref="">`, but it doesn't resolve.